src/woo_publications/publications/api/serializers.py

```diff
@@ -376,6 +376,9 @@ def update(self, instance, validated_data):
                 user={"identifier": user_id, "display_name": user_repr}, remarks=remarks
             )
 
+        if validated_data.get("informatie_categorieen"):
+            publication.apply_retention_policy()
```

Comment:
aren't these always going to be provided in PUT calls? so that would negate manual overwrites from GPP-app

Reply:
The informatie_categorieen is atm a required field so it will always be provided in the PUT call. But when you are using a patch call this isn't the case, so we don't want to accidentally trigger it in those cases.

Reply:
yeah but I don't think gpp-app uses PATCH calls. Functionally, if I make a PUT call where I change just the description of a publication, this should not re-trigger retention policy calculation. It's entirely valid to use a PUT call where you change the archiefactiedatum manually, no? Even though the UI doesn't enable this right now, I think this behaviour needs to be covered in the API now because changing this later will be a breaking change and require a new major version.